packages/docs-reanimated/docs/advanced/useHandler.mdx

```diff
@@ -4,48 +4,111 @@ sidebar_position: 5
 
 # useHandler
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+`useHandler` is a low-level hook. It returns a context object and a value that tells you if the worklet needs to be rebuilt. You can use it to create custom event handler hooks, like `useAnimatedGestureHandler` or `useAnimatedScrollHandler`.
```

Comment:
I wouldn't mention `useAnimatedGestureHandler` - it is deprecated. Let's switch it to, for example, `useScrollViewOffset` or `useComposedEventHandler`

Reply:
Also, maybe, let's link them up as well!